### DISCUSSION TITLE
How to control default mouseOver behaviours?

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14061] [Q&A] [answered]
I have a button . It has foreground and background colours. These colours change on mouseOver. I don't want this to happen. How do I control this behaviour?

The button is defined as 
```xml
        <Button x:Name="butGo" AutomationProperties.Name="Run Code" ToolTipService.ToolTip="Run Code"
                RelativePanel.RightOf="butToggleOutput" RelativePanel.Below="menuBar" Click="RunCodeButton_Click" Background="DarkGreen" Foreground="White">
            <Button.Content>
                <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE73E;" FontWeight="ExtraBold" />
            </Button.Content>
        </Button>
```
I have a RichEditBox with similar issues. The box has a coloured background but the colour disappears on mouseOver.
_LATER_
Actually, it affects other things too. I have a tabbed interface and the content of the tags on the tabs also disappear on mouseOver. Not helpful.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! To change this behavior, the easier way is to change the template of the button. You can create your own style, like this: https://playground.platform.uno/#34b5be65

If you want to keep more the original style, you can copy its contents in your app.xaml resources, and tweak it:

https://github.com/unoplatform/uno/blob/2af7b1a6953cc501dd6b16c07c4510fd8a937828/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/Button_themeresources.xaml#L140-L266

**Reply by axtens** [reply]
So does this work also for other controls? This same issue is affecting a RichEditBox and a number of other controls.
